include/caf/events/sensor_event.h

```diff
@@ -43,7 +43,10 @@ enum sensor_state {
 	SENSOR_STATE_ERROR,
 
 	/** Number of sensor states. */
-	SENSOR_STATE_COUNT
+	SENSOR_STATE_COUNT,
+
+	/** Unused in code, required for inter-core compatibility  */
+	SENSOR_STATE_FORCE_INT_SIZE = INT_MAX
```

Comment:
shouldn't it be UINT_MAX or better UINT32_MAX?

Reply:
Right - we should use defined type. Just UINT still provides point where incompatibility may appear.